lib/resources/r4/communication.yaml

```diff
@@ -0,0 +1,193 @@
+---
+name: Communication
```

Comment:
Going to be adding something for Category.... I would say we can just create one now, link it to what it is right now. And then just update the list when we figure out the other system on Thursday

Reply:
Stole the update from @VelocityKnight's PR: https://github.com/cerner/fhir.cerner.com/pull/433/commits/c7224aa383bfb23c92702f8d28bf47d5cc6dd084